integration/fixtures_test.go

```diff
@@ -1,3 +1,11 @@
 package integration_test
 
-var testCatalogSourceImage = "quay.io/operatorhubio/catalog:latest"
+import "time"
+
+var (
+	referenceAddonCatalogSourceImageWorking = "quay.io/osd-addons/reference-addon-index@sha256:58cb1c4478a150dc44e6c179d709726516d84db46e4e130a5227d8b76456b5bd"
```

Comment:
Should have a doc block what version of the reference-addon is referenced here to make it easier to follow what's going on.

Reply:
good catch! added this as well